packages/amplify-category-hosting/lib/S3AndCloudFront/helpers/file-uploader.js

```diff
@@ -6,29 +6,24 @@ const sequential = require('promise-sequential');
 const fileScanner = require('./file-scanner');
 const constants = require('../../constants');
 
-const category = 'hosting';
 const serviceName = 'S3AndCloudFront';
 const providerName = 'awscloudformation';
 
 async function run(context, distributionDirPath) {
   const { WebsiteConfiguration } = context.exeInfo.template.Resources.S3Bucket.Properties;
-  const { output } = context.exeInfo.amplifyMeta[category][serviceName];
   const fileList = fileScanner.scan(context, distributionDirPath, WebsiteConfiguration);
 
   const uploadFileTasks = [];
   const s3Client = await getS3Client(context, 'update');
   const hostingBucketName = getHostingBucketName(context);
 
-  let cloudFrontS3CanonicalUserId;
-  if (context.exeInfo.template.Resources.OriginAccessIdentity && output.CloudFrontOriginAccessIdentity) {
-    const cloudFrontClient = await getCloudFrontClient(context, 'retrieve');
-    const params = { Id: output.CloudFrontOriginAccessIdentity };
-    const originAccessIdentity = await cloudFrontClient.getCloudFrontOriginAccessIdentity(params).promise();
-    cloudFrontS3CanonicalUserId = originAccessIdentity.S3CanonicalUserId;
+  let hasCloudFront = false;
+  if (context.exeInfo.template.Resources.CloudFrontDistribution) {
+    hasCloudFront = true;
```

Comment:
let hasCloudFront = context.exeInfo.template.Resources.CloudFrontDistribution ? true : false;
```suggestion
let hasCloudFront = context.exeInfo.template.Resources.CloudFrontDistribution ? true : false;
```

Reply:
It's obviously a fine suggestion but if I commit it and reset the reviews/approvals will that set the PR back another 8 months?